Sources/XCRemoteCache/Shell/Shell.swift

```diff
@@ -84,13 +84,13 @@ private func shellInternal(_ cmd: String, args: [String] = [], stdout: PipeLike?
         task.currentDirectoryPath = dir
     }
     task.launch()
+    let errorData = (stderr == nil) ? nil : errorHandle.fileHandleForReading.readDataToEndOfFile()
```

Comment:
Shouldn't that leave the `errorData` nil, if custom `stderr` is passed and we cannot inspect it? Only if `stderr` is `nil`, we use our own pipe and then can capture errors.
 
```suggestion
    let errorData = (stderr != nil) ? nil : errorHandle.fileHandleForReading.readDataToEndOfFile()
```

Reply:
Yeah, true - sorry, I did not check it.